games-strategy/maxr/maxr-0.2.9.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+inherit cmake-utils eutils
+
+DESCRIPTION="Mechanized Assault and Exploration Reloaded"
+HOMEPAGE="http://www.maxr.org/"
+SRC_URI="http://www.maxr.org/downloads/${P}.tar.gz"
+
+LICENSE="GPL-2 FDL-1.2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="mirror"
```

Comment:
Why would this need mirror restrictions?

Reply:
removed